exchanges/kucoin/kucoin_test.go

```diff
@@ -1347,624 +1399,645 @@ func TestPostFuturesOrder(t *testing.T) {
+	result, err := ku.CancelFuturesOrderByOrderID(context.Background(), "5bd6e9286d99522a52e458de")
+	require.NoError(t, err)
+	assert.NotNil(t, result)
+}
 
-	_, err := ku.CancelFuturesOrder(context.Background(), "5bd6e9286d99522a52e458de")
-	if err != nil {
-		t.Error("CancelFuturesOrder() error", err)
-	}
+func TestCancelFuturesOrderByClientOrderID(t *testing.T) {
+	t.Parallel()
+	_, err := ku.CancelFuturesOrderByClientOrderID(context.Background(), "")
+	require.ErrorIs(t, err, order.ErrOrderIDNotSet)
+
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, ku, canManipulateRealOrders)
+	result, err := ku.CancelFuturesOrderByClientOrderID(context.Background(), "5bd6e9286d99522a52e458de")
+	require.NoError(t, err)
+	assert.NotNil(t, result)
 }
 
 func TestCancelAllFuturesOpenOrders(t *testing.T) {
 	t.Parallel()
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, ku, canManipulateRealOrders)
-
-	_, err := ku.CancelAllFuturesOpenOrders(context.Background(), "XBTUSDM")
-	if err != nil {
-		t.Error("CancelAllFuturesOpenOrders() error", err)
-	}
+	result, err := ku.CancelMultipleFuturesLimitOrders(context.Background(), "XBTUSDM")
+	require.NoError(t, err)
+	assert.NotNil(t, result)
 }
 
 func TestCancelAllFuturesStopOrders(t *testing.T) {
 	t.Parallel()
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, ku, canManipulateRealOrders)
-	_, err := ku.CancelAllFuturesStopOrders(context.Background(), "XBTUSDM")
-	if err != nil {
-		t.Error("CancelAllFuturesStopOrders() error", err)
-	}
+	result, err := ku.CancelAllFuturesStopOrders(context.Background(), "XBTUSDM")
+	require.NoError(t, err)
+	assert.NotNil(t, result)
 }
 
 func TestGetFuturesOrders(t *testing.T) {
 	t.Parallel()
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, ku)
-	_, err := ku.GetFuturesOrders(context.Background(), "", "", "", "", time.Time{}, time.Time{})
-	if err != nil {
-		t.Error("GetFuturesOrders() error", err)
-	}
+	result, err := ku.GetFuturesOrders(context.Background(), "", "", "", "", time.Time{}, time.Time{})
+	require.NoError(t, err)
+	assert.NotNil(t, result)
 }
 
 func TestGetUntriggeredFuturesStopOrders(t *testing.T) {
 	t.Parallel()
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, ku)
-	_, err := ku.GetUntriggeredFuturesStopOrders(context.Background(), "", "", "", time.Time{}, time.Time{})
-	if err != nil {
-		t.Error("GetUntriggeredFuturesStopOrders() error", err)
-	}
+	result, err := ku.GetUntriggeredFuturesStopOrders(context.Background(), "", "", "", time.Time{}, time.Time{})
+	require.NoError(t, err)
+	assert.NotNil(t, result)
 }
 
 func TestGetFuturesRecentCompletedOrders(t *testing.T) {
 	t.Parallel()
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, ku)
-	_, err := ku.GetFuturesRecentCompletedOrders(context.Background())
-	if err != nil {
-		t.Error("GetFuturesRecentCompletedOrders() error", err)
-	}
+	result, err := ku.GetFuturesRecentCompletedOrders(context.Background(), "BTCUSDT")
+	require.NoError(t, err)
+	assert.NotNil(t, result)
```

Comment:
```
        	Error:      	Received unexpected error:
        	            	code: 100003 message: Contract parameter invalid.
        	Test:       	TestGetFuturesRecentCompletedOrders
```
Probably best to stick with the FuturesTradablePair. Please do this across the string pair params I think.